app/src/sharedTest/java/org/oppia/android/app/customview/interaction/MathExpressionInteractionsViewTest.kt

```diff
@@ -1629,24 +1740,31 @@ class MathExpressionInteractionsViewTest {
   private fun launchForNumericExpressions(
     interaction: Interaction = Interaction.getDefaultInstance(),
     translationContext: WrittenTranslationContext = WrittenTranslationContext.getDefaultInstance()
-  ): ActivityScenario<InputInteractionViewTestActivity> {
+  ): ActivityScenario<MathExpressionInteractionsViewTestActivity> {
     return launch(MathInteractionType.NUMERIC_EXPRESSION, interaction, translationContext)
   }
 
   private fun launchForMathEquations(
     interaction: Interaction = Interaction.getDefaultInstance(),
     translationContext: WrittenTranslationContext = WrittenTranslationContext.getDefaultInstance()
-  ): ActivityScenario<InputInteractionViewTestActivity> {
+  ): ActivityScenario<MathExpressionInteractionsViewTestActivity> {
     return launch(MathInteractionType.MATH_EQUATION, interaction, translationContext)
   }
 
+  private fun launchForAlgebraicExpressions(
+    interaction: Interaction = Interaction.getDefaultInstance(),
+    translationContext: WrittenTranslationContext = WrittenTranslationContext.getDefaultInstance()
+  ): ActivityScenario<MathExpressionInteractionsViewTestActivity> {
+    return launch(MathInteractionType.ALGEBRAIC_EXPRESSION, interaction, translationContext)
+  }
+
   private fun launch(
     interactionType: MathInteractionType,
     interaction: Interaction = Interaction.getDefaultInstance(),
     translationContext: WrittenTranslationContext = WrittenTranslationContext.getDefaultInstance()
-  ): ActivityScenario<InputInteractionViewTestActivity> {
+  ): ActivityScenario<MathExpressionInteractionsViewTestActivity> {
     return ActivityScenario.launch(
-      InputInteractionViewTestActivity.createIntent(
+      MathExpressionInteractionsViewTestActivity.createIntent(
         ApplicationProvider.getApplicationContext(),
         InputInteractionViewTestActivityParams.newBuilder().apply {
```

Comment:
Update to MathExpressionInteractionViewTestActivityParams

Reply:
Done